src/server/JasmineGraphServer.cpp

```diff
@@ -2841,4 +2869,31 @@ void JasmineGraphServer::duplicateCentralStore(std::string graphID) {
             server_logger.log("Error reading from socket", "error");
         }
     }
+}
+
+void JasmineGraphServer::logLoadAverage(std::string name) {
```

Comment:
We should have better design for code than this. Now we have cluttered JasmineGraphServer

Reply:
Let's make it configurable. When the configuration is enabled this thread spawns up. What do you think in that approach?

Reply:
Refactored this code as discussed Dr Miyuru

Reply:
The package name for metrics should be `performance/metrics` not `performance/matrix`

Reply:
Changed the package name to metrics

Reply:
Ok